### ISSUE TITLE
Add force option when database not empty with bookshelf

### ISSUE THREAD

**Issue by chaitanyya** [strapi/strapi#2432]
**Informations**
- **Node.js version**: v10.14.1 
- **NPM version**: v6.4.1
- **Strapi version**: 3.0.0-alpha.16 
- **Database**: Postgres
- **Operating system**: 64bit Amazon Linux/2.12.4

**What is the current behavior?**
Currently if there's existing data in the database used by Strapi the application creation process is blocked by:

```bash
It seems that your database is not empty. Be aware that Strapi is going to automatically creates tables & columns, and might update columns which can corrupt data or cause data loss.

Are you sure you want to continue with the <DB_NAME> database: (Y/n)
```

This is blocking for somebody how's using an automated process to setup the application. It'd be nice if we can have an option flag which can be used when the process is blocked by the interactive command.

**Steps to reproduce the problem**
1. Use database config which already contains some data.
2. Run the following command to set up a new Strapi application.
```bash
strapi new ${APP_NAME} --dbclient=$DATABASE_CLIENT --dbhost=$DATABASE_HOST --dbport=$DATABASE_PORT --dbname=$DATABASE_NAME --dbusername=$DATABASE_USERNAME --dbpassword=$DATABASE_PASSWORD --dbssl=$DATABASE_SSL
```
3. The process is blocked by an interactive input statement.

**What is the expected behavior?**
The CLI provides an option flag to set the default value for the blocking interactive statement and uses it when setting up the application.

**Suggested solutions**
An option flag `--force` or something similar to proceed with value `Yes` instead of blocking the application setup. 

I'd like to contribute on this one, would like to know your thoughts first.

**Comment by lauriejim** [comment]
Thank you @chaitanyya for this report.
You can provide a PR to add this force option.

Check in `strapi-hook-bookshelf` package the `connection.js` file.

Check out the contributing guide to get started: https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md

**Comment by chaitanyya** [comment]
I started with the implementation and made the necessary changes to the following files:
1. `packages/strapi/bin/strapi-new.js`
2. `packages/strapi/bin/strapi.js`
3. `packages/strapi-hook-bookshelf/lib/utils/connectivity.js`

In order to test the changes I built the project using `npm run setup` and tried running the `strapi new <ALL OPTIONS>` command.

For some reason none of the changes in the `/connectivity.js` file are reflected when running `strapi new` not even the basic `console.log` commands. Am I doing something wrong?

**Comment by lauriejim** [comment]
Ho yes the process to test `connection` feature is a little bit tricky actually.

Let me explain the process: https://github.com/strapi/strapi/blob/master/packages/strapi-generate-new/lib/before.js#L220

When you create a new app, we create a temporary folder and install the database hook you need.
And the we require this installed node modules to test the database connection.

So you have to use something like `verdaccio` to deploy your hook.
But it's not an easy process.

I suggest you to update the path of the `connectivity` https://github.com/strapi/strapi/blob/master/packages/strapi-generate-new/lib/before.js#L259 file we require.

An option to let it be easier to the future is to define the path depending of `scope.developerMode`.

Maybe you can add this modification for the community.

> Hope you understand my message - little bit complicated.

**Comment by chaitanyya** [comment]
Thanks for the update. @lauriejim 

> I suggest you to update the path of the connectivity

So, if I understand it correctly instead of using the temp path I should use the path to the files in the repository.

> An option to let it be easier to the future is to define the path depending of scope.developerMode.

In order to make it easier for future development this path can be defined based on the options of the `strapi new`

I hope this makes sense.

**Comment by chaitanyya** [comment]
Okay I was able to test it by changing the path of the `connectivity`. However it did require me to include additional dependencies in the package `Knex` and `pg`.

So, if we'd want to make this path dynamic based on `scope.developerMode`. How do you think is the best way to do it?

Also, I believe it should be tracked on a separate issue and I can keep posting the updates there.

**Comment by chaitanyya** [comment]
Thanks, also I'm not sure about the test coverage. If you have any suggestions on making it more robust, feedback would be nice.

**Comment by lauriejim** [comment]
We actually haven't coverage for this part of Strapi  (not good)
If you want to check our test you have all information in `./test` folder.
We are just testing generated APIs and the content manager (focus on relations)

**Comment by chaitanyya** [comment]
The pull request is now merged (#2534) and the feature should be usable. Thanks guys :)

**Comment by lauriejim** [comment]
Your welcome! Happy to see the community contribute to Strapi!
